conda-store-server/Dockerfile

```diff
@@ -39,28 +41,41 @@ RUN printf 'channels: []\n' > /opt/conda/.condarc && \
 
 USER 1000:1000
 
-COPY environment.yaml /opt/conda-store-server/environment.yaml
+RUN mamba create --name ${conda_env_name} \
+    python=${python_version} conda \
+    --channel conda-forge -y && \
+    conda clean --force-pkgs-dirs --all -y
 
-RUN mamba env create -f /opt/conda-store-server/environment.yaml && \
-	conda clean  --force-pkgs-dirs
+# ensure we are using the conda environment
+ENV PATH=/opt/conda/condabin:/opt/conda/envs/conda-store-server/bin:/opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${PATH}
```

Comment:
Hmm, why do we need this now? Was `PATH` not correctly set in the previous dockerfile?

Reply:
I just moved its placement so it was the same for both Dockerfiles; I can move these both to the top if that is more intuitive

Reply:
nvm, moved it so we have all the envs together at the top